src/main/java/io/confluent/idesidecar/restapi/models/ConnectionSpec.java

```diff
@@ -49,7 +51,17 @@ public record ConnectionSpec(
                           + "Apache Kafka cluster.")
     @JsonProperty(KAFKA_CLUSTER_CONFIG_FIELD_NAME) KafkaClusterConfig kafkaClusterConfig,
     @Schema(description = "The details for connecting to a Schema Registry.")
-    @JsonProperty(SCHEMA_REGISTRY_CONFIG_FIELD_NAME) SchemaRegistryConfig schemaRegistryConfig
+    @JsonProperty(SCHEMA_REGISTRY_CONFIG_FIELD_NAME) SchemaRegistryConfig schemaRegistryConfig,
+    @Schema(
+        description =
+            "Whether to communicate with the cluster over TLS/SSL. Defaults to 'true', "
+                + "but set to 'false' when the cluster does not support TLS/SSL.",
+        defaultValue = KafkaClusterConfig.DEFAULT_SSL_VALUE,
```

Comment:
The `TLSConfig tlsConfig` parameter is not a boolean, so the `Defaults to ...` and `defaultValue` here are not right.